src/Reflection/Adapter/ReflectionClass.php

```diff
@@ -180,7 +180,7 @@ public function hasProperty($name)
     /**
      * {@inheritDoc}
      */
-    public function getProperty($name)
+    public function getProperty($name) : ?ReflectionProperty
```

Comment:
AFAIK, all adapters didn't really have return types. Wondering if we should change that...

Reply:
I can put it in the docblock?

Reply:
PHP *allows* it...